### ISSUE TITLE
OverlayView: Target container is not a DOM element.

### ISSUE THREAD

**Issue by hasyee** [tomchentw/react-google-maps#167]
```
<OverlayView
	position={{lat: 47, lng: 19}}
	mapPaneName={OverlayView.OVERLAY_LAYER}
	getPixelPositionOffset={(width, height) => ({x: -(width / 2), y: -(height / 2)})}
>
	<div style={{backgroundColor: "white"}}></div>
</OverlayView>
```

![issue](https://cloud.githubusercontent.com/assets/7387621/11511367/062ae8a8-9869-11e5-8e74-d88a8df334ea.PNG)

**Comment by tomchentw** [comment]
Does this issue still exist?

**Comment by ohardy** [comment]
@tomchentw got it too on last version

**Comment by tomchentw** [comment]
Released [v4.7.2](https://github.com/tomchentw/react-google-maps/releases/tag/v4.7.2).
